docs/guide/time.rst

```diff
@@ -16,9 +18,41 @@ SunPy builds upon its functionality.
 
 Solar data is associated with a number of different time formats. SunPy provides a simple
 parsing function which can deal with most every format that a user may encounter. Called
-`sunpy.time.parse_time()`, this function takes a string as input and returns a `~astropy.time.Time` object.
-
-Here are few examples of formats which `sunpy.time.parse_time()` accepts::
+`sunpy.time.parse_time()`, this function can take a variety of inputs.
+
+strings
+^^^^^^^
+
+The most commonly used are strings and we support a selection of formats
+which are matched using regrex. We currently support the following style of string formats::
+
+    "2007-05-04T21:08:12.999999"
```

Comment:
I like having this list, I do wonder if it would be possible to autogenerate it though?

Reply:
Well I wanted to add the regex and time format variables in the docs but I couldn't work out a way. So I just did this for now.